app/models/jubla/group.rb

```diff
@@ -85,4 +85,10 @@ def alumni_filter_chain
       role_types: self.class.roles.collect(&:sti_name)
     }}
   end
+
+  def require_person_add_requests
+    true
+  end
+
+  alias_method :require_person_add_requests?, :require_person_add_requests
```

Comment:
Braucht es das tatsächlich, wenn wir das Setting in der DB verändern und activate/deactivate verhindern? Um das Default zu setzen, damit es auch neue Gruppe betrifft, würde ich lieber einen `before_create`-hook sehen, der das Attribut entsprechend setzt.

Reply:
Ich habe es auf die DB Lösung geändert, anstatt eine `before_create` hook zu verwenden, habe ich den default value der db spalte direkt angepasst.

Reply:
Hast du probiert, ob eine neue Gruppe dann auch das Attribute richtig gesetzt hat? Das DB-Default greift ja nur, wenn Rails nichts sendet. Und genau an dem Punkt vertraue ich Rails nicht wirklich... Daher der `before_create`-hook, damit sichergestellt wird, dass Rails auch das richtige an die DB sendet.